contrib/znc/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+# create znc datadir
+
+d /var/lib/znc 0755 znc znc -
```

Comment:
you call this user/group `znc` but you make `_znc` above so this doesn't exist, should be _znc _znc

Reply:
and probably 0750